manage

```diff
@@ -16,18 +16,22 @@ export LEDGER_TIMEOUT=60
 # these two can be overrode via env vars
 export LEDGER_URL_CONFIG="${LEDGER_URL_CONFIG}"
 export TAILS_SERVER_URL_CONFIG="${TAILS_SERVER_URL_CONFIG}"
+export REDIS_CLUSTER_URL_CONFIG="${REDIS_CLUSTER_URL_CONFIG}"
 export AGENT_CONFIG_FILE="${AGENT_CONFIG_FILE}"
 
 # these are derived from the above two
-LEDGER_URL_HOST="${LEDGER_URL_CONFIG:-http://localhost:9000}"
-LEDGER_URL_INTERNAL="${LEDGER_URL_CONFIG:-http://${DOCKERHOST}:9000}"
+LEDGER_URL_HOST="${LEDGER_URL_CONFIG:-http://localhost:9001}"
```

Comment:
OK -- that was easy.  This is the problem.  Why the change to `9001` here?  von-network always runs at 9000.  A quick change and it worked...

Reply:
Thanks.... my machine has port 9000 occupied, so I've had to keep local changes that change it to 9001, we should try and make it configurable eventually.